internal/services/compute/virtual_machine_scale_set_resource.go

```diff
@@ -1942,7 +1947,7 @@ func expandazurestackVirtualMachineScaleSetsStorageProfileDataDisk(d *pluginsdk.
 	return &dataDisks
 }
 
-func expandazurestackVirtualMachineScaleSetStorageProfileImageReference(d *pluginsdk.ResourceData) (*compute.ImageReference, error) {
+func expandAzureRmVirtualMachineScaleSetStorageProfileImageReference(d *pluginsdk.ResourceData) (*compute.ImageReference, error) {
```

Comment:
could we change all these AzureRm -> 
```suggestion
func expandVirtualMachineScaleSetStorageProfileImageReference(d *pluginsdk.ResourceData) (*compute.ImageReference, error) {
```

Reply:
Good eye, will update the naming, thank you